src/Microsoft.ML.Core/Data/RoleMappedSchema.cs

```diff
@@ -534,7 +534,7 @@ public RoleMappedData(IDataView data, IEnumerable<KeyValuePair<RoleMappedSchema.
         /// <param name="opt">Whether to consider the column names specified "optional" or not. If <c>false</c> then any non-empty
         /// values for the column names that does not appear in <paramref name="data"/>'s schema will result in an exception being thrown,
         /// but if <c>true</c> such values will be ignored</param>
-        public RoleMappedData(IDataView data, string label, string feature,
+        public RoleMappedData(IDataView data, string label, string feature = null,
```

Comment:
> = null [](start = 74, length = 7)

undo this

Reply:
I will handle this in my code then! Sorry about that

---
In reply to: [218244002](https://github.com/dotnet/machinelearning/pull/876#discussion_r218244002) [](ancestors = 218244002)